Modules/Challenges/test/Challenges.Application.Tests/Tests/Challenges/Commands/DeleteChallengesOfIdentity/ValidatorTests.cs

```diff
@@ -0,0 +1,40 @@
+﻿using Backbone.Modules.Challenges.Application.Challenges.Commands.DeleteChallengesOfIdentity;
+using Backbone.UnitTestTools.BaseClasses;
+using Backbone.UnitTestTools.Data;
+using Backbone.UnitTestTools.FluentValidation;
+using FluentValidation.TestHelper;
+using Xunit;
+
+namespace Backbone.Modules.Challenges.Application.Tests.Tests.Challenges.Commands.DeleteChallengesOfIdentity;
+
+public class ValidatorTests : AbstractTestsBase
+{
+    [Fact]
+    public void Happy_path()
+    {
+        // Arrange
+        var validator = new Validator();
+
+        // Act
+        var validationResult = validator.TestValidate(new DeleteChallengesOfIdentityCommand(TestDataGenerator.CreateRandomIdentityAddress()));
+
+        // Assert
+        validationResult.ShouldNotHaveAnyValidationErrors();
+    }
+
+    [Fact]
+    public void Fails_when_identity_address_is_invalid()
+    {
+        // Arrange
+        var validator = new Validator();
+
+        // Act
+        var validationResult = validator.TestValidate(new DeleteChallengesOfIdentityCommand("invalid-identity-address"));
+
+        // Assert
+        validationResult.ShouldHaveValidationErrorForItem(
+            propertyName: nameof(DeleteChallengesOfIdentityCommand.IdentityAddress),
+            expectedErrorCode: "error.platform.validation.invalidPropertyValue",
+            expectedErrorMessage: "The ID is not valid. Check length, prefix and the used characters.");
```

Comment:
Since the error code and message are always the same, your current solution is unnecessarily verbose, imo. You might as well do this:

```suggestion
        validationResult.ShouldHaveValidationErrorDueToInvalidId(propertyWithInvalidId: nameof(DeleteChallengesOfIdentityCommand.IdentityAddress));
```

Reply:
It's not always the same, but I created a couple extension methods to avoid repetition of the error messages and codes.